src/games/strategy/engine/framework/startup/mc/GameSelectorModel.java

```diff
@@ -250,20 +233,18 @@ public synchronized ClientModel getClientModelForHostBots() {
   }
 
   /**
-   * We dont have a gane data (ie we are a remote player and the data has not been sent yet), but
-   * we still want to display game info
+   * We dont have a gane data (ie we are a remote player and the data has not been sent yet), but we
+   * still want to display game info
    */
   public void clearDataButKeepGameInfo(final String gameName, final String gameRound, final String gameVersion) {
-    synchronized (this) {
```

Comment:
This is perhaps the only synchronize block that makes sense, to ensure that gameName and gameRound and gameVersion will be set consistenly.  If you look at callers of this method, we do not have a real risk of that.